types/test/toArray.ts

```diff
@@ -1,7 +1,10 @@
 import * as RA from 'ramda-adjunct';
 
-RA.toArray([1, 2, 3]); // $ExpectType any[]
-RA.toArray(null); // $ExpectType any[]
-RA.toArray(undefined); // $ExpectType any[]
-RA.toArray(1); // $ExpectType any[]
-RA.toArray({}); // $ExpectType any[]
+RA.toArray('1'); // $ExpectType string[]
+RA.toArray([1, 2, 3]); // $ExpectType number[]
+RA.toArray({ a: 1, b: 2 }); // $ExpectType number[]
+RA.toArray({}); // $ExpectType never[]
+
+RA.toArray(null); // $ExpectError
```

Comment:
Any expecting error here? I'd say null is valid input, because the function returns empty array.

Reply:
Hmm, I'd say if we go for this direction, we should change the method signature, because [you can't pass `null` where you expect an `Iterable`](https://www.typescriptlang.org/play/#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwxwEEYYoBPAHgBUA+ACiwxDICMIQAueEs8gGSwBrEDVrwAPvACSzNhzEBKbtQDaAXQDcAKG2FeFeqmQQIizUA).

Reply:
Right, let's not go in this direction. Disregard my comment.